integ-test/src/test/java/org/opensearch/sql/multicluster/OpenSearchMultiClustersRestTestCase.java

```diff
@@ -0,0 +1,137 @@
+  public void initRemoteClient() throws IOException {
+    if (remoteClient == null) {
+      assert remoteAdminClient == null;
+      String cluster = getTestRestCluster(REMOTE_CLUSTER);
+      String[] stringUrls = cluster.split(",");
+      List<HttpHost> hosts = new ArrayList<>(stringUrls.length);
+      for (String stringUrl : stringUrls) {
+        int portSeparator = stringUrl.lastIndexOf(':');
+        if (portSeparator < 0) {
+          throw new IllegalArgumentException("Illegal cluster url [" + stringUrl + "]");
+        }
+        String host = stringUrl.substring(0, portSeparator);
+        int port = Integer.valueOf(stringUrl.substring(portSeparator + 1));
+        hosts.add(buildHttpHost(host, port));
+      }
+      final List<HttpHost> clusterHosts = unmodifiableList(hosts);
+      logger.info("initializing REST clients against {}", clusterHosts);
+      remoteClient = buildClient(restClientSettings(), clusterHosts.toArray(new HttpHost[0]));
+      remoteAdminClient = buildClient(restAdminSettings(), clusterHosts.toArray(new HttpHost[0]));
+    }
+    assert remoteClient != null;
+    assert remoteAdminClient != null;
+  }
+
+  /**
+   * Get a comma delimited list of [host:port] to which to send REST requests.
+   */
+  protected String getTestRestCluster(String clusterName) {
+    String cluster = System.getProperty("tests.rest." + clusterName + ".http_hosts");
+    if (cluster == null) {
+      throw new RuntimeException(
+          "Must specify [tests.rest."
+              + clusterName
+              + ".http_hosts] system property with a comma delimited list of [host:port] "
+              + "to which to send REST requests"
+      );
+    }
+    return cluster;
+  }
+
+  /**
+   * Get a comma delimited list of [host:port] for connections between clusters.
+   */
+  protected String getTestTransportCluster(String clusterName) {
+    String cluster = System.getProperty("tests.rest." + clusterName + ".transport_hosts");
+    if (cluster == null) {
+      throw new RuntimeException(
+          "Must specify [tests.rest."
+              + clusterName
+              + ".transport_hosts] system property with a comma delimited list of [host:port] "
+              + "for connections between clusters"
+      );
+    }
+    return cluster;
+  }
+
+  /**
+   * Initialize rest client to remote cluster,
+   * and create a connection to it from the coordinating cluster.
+   */
+  public void configureMultiClusters() throws IOException {
+    initRemoteClient();
+
+    Request connectionRequest = new Request("PUT", "_cluster/settings");
+    String connectionSetting = "{\"persistent\": {\"cluster\": {\"remote\": {\""
+        + REMOTE_CLUSTER
+        + "\": {\"seeds\": [\""
+        + getTestTransportCluster(REMOTE_CLUSTER).split(",")[0]
+        + "\"]}}}}}";
+    connectionRequest.setJsonEntity(connectionSetting);
+    logger.info("Creating connection from coordinating cluster to {}", REMOTE_CLUSTER);
```

Comment:
for debug?

Reply:
no. just to leave some log.
It's like [this](https://github.com/opensearch-project/OpenSearch/blob/d984f5028e77e281ed08386d43de4e64e06f2bf8/test/framework/src/main/java/org/opensearch/test/rest/OpenSearchRestTestCase.java#L209) for `initClient` in `OpenSearchRestTestCase`

Reply:
update: removed log